docs/contributing/devguide.md

```diff
@@ -50,7 +28,7 @@ make build
 make docker.build IMG=external-secrets:latest
 ```
 
-Run tests and lint the code: *(golangci-lint@1.49.0 is needed.)*
```

Comment:
The same with golangci-lint
https://github.com/external-secrets/external-secrets/pull/2588/files#r1286483696